monkey/tests/unit_tests/monkey_island/cc/services/utils/cyphertexts_for_encryption_test.py

```diff
@@ -0,0 +1,206 @@
+MALFORMED_CYPHER_TEXT_TOO_SHORT = (
+    b"AES\x02\x00\x00\x1bCREATED_BY\x00pyAesCrypt "
+    b"6.0.0\x00\x80\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00"
+)
+
+MALFORMED_CYPHER_TEXT_CORRUPTED = (
```

Comment:
Does this belong in `tests/data_for/tests/`?

Reply:
The scope of this data is one test. If we use the same data for many different tests it means we're testing the same thing/codebase is highly coupled. `tests/data_for_tests/` reflects that something is wrong with the way the codebase handles server configs. Filling dirs like that is an unfortunate necessity, not a good practice. 

This could've been put inside the test itself, but was too big and the data itself is pretty meaningless to read, that's why I moved it.